src/platform_impl/apple/uikit/monitor.rs

```diff
@@ -76,6 +76,54 @@ pub struct MonitorHandle {
     ui_screen: MainThreadBound<Retained<UIScreen>>,
 }
 
+impl MonitorHandleProvider for MonitorHandle {
+    fn native_id(&self) -> u64 {
+        todo!()
```

Comment:
Do pointer comparison for now (see also https://github.com/rust-windowing/winit/pull/4013).

```suggestion
        // SAFETY: Only getting the pointer.
        let mtm = unsafe { MainThreadMarker::new_unchecked() };
        Retained::as_ptr(self.ui_screen.get(mtm)) as u64
```